vertx-config/src/main/java/io/vertx/config/spi/ConfigStoreFactory.java

```diff
@@ -0,0 +1,29 @@
+package io.vertx.config.spi;
+
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Factory to create instances of {@link ConfigStore}. This is a SPI, and so implementations are retrieved
+ * from the classpath / classloader using a {@link java.util.ServiceLoader}.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public interface ConfigStoreFactory {
```

Comment:
Why ? The ConfigStore has a close method, the factory itself does not really care.